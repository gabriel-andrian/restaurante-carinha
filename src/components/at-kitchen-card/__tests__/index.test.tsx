import React from 'react';
import AtKitchenCard from '../index';
import renderer from 'react-test-renderer';

describe('at-kitchen-card', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<AtKitchenCard amount={3} itemId={4} note={'item de teste'} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
