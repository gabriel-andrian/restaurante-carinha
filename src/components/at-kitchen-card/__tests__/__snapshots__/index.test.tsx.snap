// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`at-kitchen-card renders correctly 1`] = `
Array [
  <div
    className="sc-bdnylx sc-gtssRu kcPoms cgEAKG"
  >
    3
  </div>,
  <div
    className="sc-bdnylx kcPoms"
  >
    4
  </div>,
  <div
    className="sc-bdnylx sc-dlnjPT sc-hKFyIo kcPoms iHznbp hpAWnk"
  >
    Obs.:
  </div>,
  <div
    className="sc-bdnylx sc-dlnjPT kcPoms iHznbp"
  >
    item de teste
  </div>,
]
`;
